MonoGame.Framework/Desktop/OpenTKGameWindow.cs

```diff
@@ -210,39 +214,51 @@ private void Keyboard_KeyDown(object sender, OpenTK.Input.KeyboardKeyEventArgs e
 
         private void OnResize(object sender, EventArgs e)
         {
-            // Ignore resize events until intialization is complete
-            if (Game == null)
-                return;
+            lock (window)
+            {
+                if (window == null)
+                    return;
+                
+                // Ignore resize events until intialization is complete
+                if (Game == null)
+                    return;
 
-            var winWidth = window.ClientRectangle.Width;
-            var winHeight = window.ClientRectangle.Height;
-            var winRect = new Rectangle(0, 0, winWidth, winHeight);
+                var winWidth = window.ClientRectangle.Width;
+                var winHeight = window.ClientRectangle.Height;
+                var winRect = new Rectangle(0, 0, winWidth, winHeight);
 
-            // If window size is zero, leave bounds unchanged
-            // OpenTK appears to set the window client size to 1x1 when minimizing
-            if (winWidth <= 1 || winHeight <= 1) 
-                return;
+                // If window size is zero, leave bounds unchanged
+                // OpenTK appears to set the window client size to 1x1 when minimizing
+                if (winWidth <= 1 || winHeight <= 1)
+                    return;
 
-            //If we've already got a pending change, do nothing
-            if (updateClientBounds)
-                return;
+                //If we've already got a pending change, do nothing
+                if (updateClientBounds)
+                    return;
             
-            Game.GraphicsDevice.PresentationParameters.BackBufferWidth = winWidth;
-            Game.GraphicsDevice.PresentationParameters.BackBufferHeight = winHeight;
+                Game.GraphicsDevice.PresentationParameters.BackBufferWidth = winWidth;
+                Game.GraphicsDevice.PresentationParameters.BackBufferHeight = winHeight;
 
-            Game.GraphicsDevice.Viewport = new Viewport(0, 0, winWidth, winHeight);
+                Game.GraphicsDevice.Viewport = new Viewport(0, 0, winWidth, winHeight);
 
-            clientBounds = winRect;
+                clientBounds = winRect;
 
-            OnClientSizeChanged();
+                OnClientSizeChanged();
+            }
         }
 
         internal void ProcessEvents()
         {
-            UpdateBorder();
-            Window.ProcessEvents();
-            UpdateWindowState();
-            HandleInput();
+            lock (window)
+            {
+                if (window == null)
```

Comment:
Yea, you are right, but when I thought about it null check is actually useless there now because of some changes I did after I added it originally.

lock is there because resize event also modifies window so that those two don't conflict.